packages/react-tweet/src/date-utils.ts

```diff
@@ -0,0 +1,28 @@
+const options: Intl.DateTimeFormatOptions = {
+  hour: 'numeric',
+  minute: '2-digit',
+  hour12: true,
+  weekday: 'short',
+  month: 'short',
+  day: 'numeric',
+  year: 'numeric',
+}
+
+const formatter = new Intl.DateTimeFormat('en-US', options)
+
+type PartsObject = Record<keyof Intl.DateTimeFormatPartTypesRegistry, string>
```

Comment:
Can you please move the type definition at the top?

Reply:
you got it! 🫡